gNMI_Interface/Subscribe.resource

```diff
@@ -50,7 +50,7 @@ Device closes the stream
 
 Streaming subscription with mode ${stream-mode} with default encoding
     [Documentation]    Start a streaming subscription with given stream mode.
-    Subscribe    STREAM    ${lib_config.default_encoding}    ${stream-mode}
+    Subscribe    STREAM    ${lib_config.default_encoding}    ${stream-mode}    ${SAMPLE-PERIOD}
```

Comment:
number of params starts to bulk-up, maybe using some dataclass/composite object with all these properties as members could be used to simplify library API/reduce number of parameters?

Reply:
Created a separate issue, that would require somewhat larger change - not now.